### ISSUE TITLE
How to set the threshold myself ?

### ISSUE THREAD

**Issue by chenyuqing** [arunmandal53/facematch#4]
threshold = 1.10 # set yourself to meet your requirement

How to set the threshold myself ? Is there any methods to train the threshold ?

**Comment by arunmandal53** [comment]
@chenyuqing it should be set manually if you increase it you may have false positive if you decrease it you will be accurate. If your requirement is to compare two photos of a person with difference in age, angle, pose then increase the threshold will help but may have false positive for other images.

**Comment by chenyuqing** [comment]
@arunmandal53 So grateful for your answer. I just want to do the face verification. So what is the range of the threshold. And if I want more accuracy on my dataset, should I just decrease it ?

**Comment by arunmandal53** [comment]
@chenyuqing yes, for face verification provided that the face is directly faced toward camera and straight then decreasing threshold is good for full accuracy. Use near to 0.5

**Comment by chenyuqing** [comment]
@AntonOellerer Thansk for your suggestion. But it doesn't work well when I test it on me and my colleague. It will treat us as the same person. Which is not so good. So how can I train the model with my own data. How to set up threshold when I am training? Is there any strict rule for setting the threshold. Which confused me so long. Thanks!!

**Comment by chenyuqing** [comment]
@AntonOellerer Thansk for your suggestion. But it doesn't work well when I test it on me and my friend. It will treat us as the same person. Which is not so good. So how can I train the model with my own data. How to set up threshold when I am training? Is there any strict rule for setting the threshold. Which confused me so long. Thanks!!

**Comment by chenyuqing** [comment]
@arunmandal53  Thansk for your suggestion. But it doesn't work well when I test it on me and my friend. It will treat us as the same person. Which is not so good. So how can I train the model with my own data. How to set up threshold when I am training? Is there any strict rule for setting the threshold. Which confused me so long. Thanks!!

**Comment by chenyuqing** [comment]
@arunmandal53 Also, can I ask you the range of threshold in your project ? Thanks!

**Comment by Ashish-Surve** [comment]
@chenyuqing 
Check this out, I have created a fork and made the project executable and added a few features including face recognition of custom data set. Check it out,
[Link to github](https://github.com/Ashish-Surve/facematch/blob/master/README.md)
